src/views/Authentication/SyringeIllustartion.tsx

```diff
@@ -0,0 +1,256 @@
+import { Icon, IconProps } from '@chakra-ui/react';
+
+export const SyringeIllustartion = (props: IconProps) => {
+  return (
+    <Icon
+      width='100%'
+      viewBox='0 0 500 500'
+      fill='none'
+      xmlns='http://www.w3.org/2000/svg'
+      {...props}
+    >
+      <defs>
+        <clipPath id='a' clipPathUnits='userSpaceOnUse'>
+          <path d='M0 595.276h881.541V0H0Z' />
+        </clipPath>
+      </defs>
+      <g
+        clip-path='url(#a)'
+        transform='matrix(1.33333 0 0 -1.29191 -548.31 640.58)'
+      >
+        <path
```

Comment:
I wouldn't put SVG in a TSX file like this, please use an `<img>` tag and load it as a file.

Reply:
This is the way on how to do it to use the chakra icon classes. Normally I would do it like you said. Any suggestions why not to use chakra for this?